### ISSUE TITLE
Make ROSS objects pickable

### ISSUE THREAD

**Issue by raphaeltimbo** [ross-rotordynamics/ross#825]
Currently we are not able to pickle ROSS objects:

```python
import ross as rs
import pickle

rotor = rs.rotor_example()
pickle.loads(pickle.dumps(rotor))


---------------------------------------------------------------------------
PicklingError                             Traceback (most recent call last)
<ipython-input-7-09fc4ffa5e19> in <module>
      3 
      4 rotor = rs.rotor_example()
----> 5 pickle.loads(pickle.dumps(rotor))

PicklingError: Can't pickle <class 'ross.rotor_assembly.GlobalIndex'>: attribute lookup GlobalIndex on ross.rotor_assembly failed


```

pickle is used by the multiprocessing module, so it would be good to have the possibilite to use it with ROSS if we want to be able to have some code running in parallel.

**Comment by ross-bott** [comment]
Hi there!
I have marked this issue as stale because it has not had activity for 45 days.
Consider the following options:
- If the issue refers to a large task, break it in smaller issues that can be solved in
less than 45 days;
- Label the issue as `wontfix` or `wontfix for now` and close it.

**Comment by raphaeltimbo** [comment]
Closed with #844.
